Lib/test/test_concurrent_futures/util.py

```diff
@@ -50,14 +51,19 @@ def setUp(self):
                 max_workers=self.worker_count,
                 mp_context=self.get_context(),
                 **self.executor_kwargs)
+            self.manager = multiprocessing.Manager()
```

Comment:
Why using `multiprocessing.Manager()` and not `self.get_context().Manager()` actually?

Reply:
I switched to `self.get_context().Manager()`.